client/src/modules/list/components/ItemsContainer.jsx

```diff
@@ -125,18 +78,15 @@ class ItemsContainer extends Component {
             {this.renderHeadingText()}
           </h2>
           <div className="items__header-controls">
-            <FilterBox
-              filterBy={filterBy}
-              label={formatMessage({ id: 'list.items-container.filter-box' })}
-              onChange={this.onFilterChange}
-              options={filterOptions}
-            />
-            <SortBox
-              label={formatMessage({ id: 'list.items-container.sort-box' })}
-              onChange={this.onSortChange}
-              options={sortOptions}
-              sortBy={sortBy}
-              sortOrder={sortOrder}
+            <Filter
+              buttonContent={<CloseIcon />}
+              classes="items__filter"
+              fields={filterByFields}
```

Comment:
hmm, `fields` prop is good because it is short but now I wonder if it is descriptive... Maybe sth like `fieldsToMatch`, `filterByFields`?